openedx_lti_tool_plugin/views.py

```diff
@@ -258,6 +259,10 @@ def post(
             log.error('LTI 1.3: Course enrollment failed: %s', exc)
             return HttpResponseBadRequest(self.BAD_RESPONSE_MESSAGE)
 
+        if not usage_key_string and not ALLOW_COMPLETE_COURSE_LAUNCH.is_enabled():
+            log.error('LTI 1.3: Complete course launches are not enabled.')
+            return HttpResponseBadRequest(self.BAD_RESPONSE_MESSAGE)
+
         if not usage_key_string:
             return redirect(f'{AppConfig.name}:lti-course-home', course_id=course_id)
```

Comment:
```suggestion
        if not usage_key_string:
            if not not ALLOW_COMPLETE_COURSE_LAUNCH.is_enabled():
                log.error('LTI 1.3: Complete course launches are not enabled.')
                return HttpResponseBadRequest(self.BAD_RESPONSE_MESSAGE)

            return redirect(f'{AppConfig.name}:lti-course-home', course_id=course_id)
```

@alexjmpb What do you think about removing the second if condition?